docs/examples/viz_slicer.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+from fury import actor, window
+from fury.utils import get_slices, show_slices
+from dipy.data import read_mni_template
+
+###############################################################################
+# Let's read the 3D data from dipy
+
+nifti = read_mni_template()
+data = np.asarray(nifti.dataobj)
+print(data.shape)
+
+###############################################################################
+# Create slice actor to visualize the 3D data as XY, YZ and XZ slices.
+
+slicer_actor = actor.slicer(data)
+scene = window.Scene()
+scene.add(slicer_actor)
+
+
+def handle_pick_event(event):
+    info = event.pick_info
+    intensity = np.asarray(info["rgba"].rgb).mean()
+    print(f"Voxel {info['index']}: {intensity:.2f}")
```

Comment:
is it always needed or just and example ?

Reply:
voxel intensity seems wrong or is it between 0 and 1 ?

Reply:
Between 0 and 1. Which is the case in Horizon as well